authors/apps/articles/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+from authors.apps.authentication.models import User
+from django.utils import text
+
+class Article(models.Model):
+    title = models.CharField(db_index=True, max_length=255)
+    description = models.TextField()
+    body = models.TextField(unique=True)
+    createdAt = models.DateTimeField(auto_now_add=True, editable=False)
+    updatedAt = models.DateTimeField(auto_now_add=True)
+    author = models.ForeignKey(User, on_delete=models.CASCADE, related_name='author')
+    slug = models.SlugField()
+    image = models.URLField(blank=True, default='ulr', max_length=200)
```

Comment:
doesnt the default have to be `default='url'` instead of `default='ulr'`

Reply:
Awwwww, I need to up my game as far as paying attention to detail is concerned. Thanks @neelxie for that observation. Let me work on it.